js/client.js

```diff
@@ -45,12 +53,13 @@ class Client {
 
   // Send a looking-for-game message.
   register() {
-    console.log("registering as " + this.name)
+    console.logVerbose("registering as " + this.name)
     this.send({op: "register", name: this.name})
   }
 
   forceUpdate() {
-    console.log("forceUpdate on the client was not overridden")
+    window.client.gameView.forceUpdate();
```

Comment:
I don't think you should muck with this - it makes the gamestate unusable without this global dependency. The injection way it was before is better